src/tools/r2rdump/R2RReader.cs

```diff
@@ -113,6 +133,41 @@ public unsafe R2RReader(string filename)
             }
         }
 
+        public static string GetRegName(Machine machine, int regNum)
```

Comment:
The format of UnwindInfo you currently support is only for AMD64. ARM / ARM64 use completely different one. So there is no need to add support for other processor registers until we have support for dumping unwind info for the other architectures.

Reply:
ok, I've removed the GetRegName function so it just uses the Registers enum for now